code/datums/voice_of_god_command.dm

```diff
@@ -0,0 +1,427 @@
+
+/datum/voice_of_god_command/getup/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	for(var/mob/living/target as anything in listeners)
+		target.set_resting(FALSE)
+		target.SetAllImmobility(0)
+
+/datum/voice_of_god_command/sit
+	trigger = "sit"
+	is_regex = FALSE
+
+/datum/voice_of_god_command/sit/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	for(var/mob/living/target as anything in listeners)
+		for(var/obj/structure/chair/chair in get_turf(target))
+			chair.buckle_mob(target)
+			break
+
+/datum/voice_of_god_command/stand
+	trigger = "stand"
+	is_regex = FALSE
+
+/datum/voice_of_god_command/stand/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	for(var/mob/living/target as anything in listeners)
+		if(target.buckled && istype(target.buckled, /obj/structure/chair))
+			target.buckled.unbuckle_mob(target)
+
+/datum/voice_of_god_command/jump
+	trigger = "jump"
+	is_regex = FALSE
+
+/datum/voice_of_god_command/jump/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	var/iteration = 1
+	for(var/mob/living/target as anything in listeners)
+		if(prob(25))
+			addtimer(CALLBACK(target, /atom/movable/proc/say, "HOW HIGH?!!"), 5 * iteration)
+		else
+			addtimer(CALLBACK(target, /mob/living/.proc/emote, "jump"), 5 * iteration)
+		iteration++
+
+/datum/voice_of_god_command/honk
+	trigger = "ho+nk"
+
+/datum/voice_of_god_command/honk/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, get_turf(user), 'sound/items/bikehorn.ogg', 300, 1), 25)
+	if(user.mind && user.mind.assigned_role == "Clown")
+		for(var/mob/living/carbon/target in listeners)
+			target.slip(140 * power_multiplier)
+
+/datum/voice_of_god_command/multispin
+	trigger = "like\\s*a\\s*record\\s*baby|right\\s*round"
+
+/datum/voice_of_god_command/multispin/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	for(var/mob/living/target as anything in listeners)
+		target.SpinAnimation(speed = 10, loops = 5)
+
+/// Supertype of all those commands who make people emote and nothing else. Fuck copypasta.
+/datum/voice_of_god_command/emote
+	/// The emote to run.
+	var/emote_name = "dance"
+
+/datum/voice_of_god_command/emote/execute(list/listeners = list(), mob/living/user, power_multiplier = 1, message)
+	var/iteration = 1
+	for(var/mob/living/target as anything in listeners)
+		addtimer(CALLBACK(target, /mob/living/.proc/emote, emote_name), 5 * iteration)
+		iteration++
+
+/datum/voice_of_god_command/emote/flip
+	trigger = "flip|rotate|revolve|roll|somersault"
+	emote_name = "flip"
+
+/datum/voice_of_god_command/emote/dance
+	trigger = "dance"
+	is_regex = FALSE
+
+/datum/voice_of_god_command/emote/salute
+	trigger = "salute"
+	emote_name = "salute"
+	is_regex = FALSE
+
+/datum/voice_of_god_command/emote/play_dead
+	trigger = "play\\s*dead"
```

Comment:
While we're here, could you add "feign death"?

Reply:
:-1: Make a different PR, refactors should not alter behavior.

Reply:
It already alters whitespace related behavior.